less/home-page.less

```diff
@@ -1,29 +1,52 @@
 // Home Content
 
-.home {
+.home, .about{
 
   .intro {
 
+    .intro-img {
+      .large-img-border-radius();
+
+      // @media @phone {
+      //   width: 100%;
+      // }
+
+    }
+
     h2 {
       padding: 15px;
     }
 
     p {
-      padding:0 15px;
+      padding: 0 15px;
     }
-  
+
   }
 
   .space-images {
-    display: flex;
+    display  : flex;
     flex-wrap: wrap;
-    width: 100%;
+    width    : 100%;
 
     img {
-      width: 28%;
-      margin: 20px 2.5%;
+      width        : 28%;
+      margin       : 20px 2.5%;
       border-radius: 20px;
-      height: 100%;
+      height       : 100%;
+    }
+
+    @media @phone {
```

Comment:
Good job setting the images to display: none; Remember you can use the :not(:first-child){ display: none;} as well.